views/explorer.tmpl

```diff
@@ -106,14 +93,14 @@
                             <span class="d-none d-md-inline">Version</span>
                             <span class="d-md-none position-relative" data-tooltip="block version">Ver</span>
                         </th>
-                        <th class="text-right" class="jsonly">Age</th>
-                        <th class="text-right px-0">Time</th>
+                        <th class="text-center" class="jsonly">Age</th>
+                        <th class="text-center px-0">Time</th>
```

Comment:
So this is an interesting problem. The mockups have this right aligned, but with the title centered over the text, but that's actually pretty hard to do with CSS.

Typically our tables always use right alignment on the last column. It helps to frame the table out. 

If you can figure out how to replicate the mockup, please do. Otherwise, leave this as right-aligned.